core/src/components/input/input.tsx

```diff
@@ -771,6 +792,30 @@ export class Input implements ComponentInterface {
                 <ion-icon aria-hidden="true" icon={mode === 'ios' ? closeCircle : closeSharp}></ion-icon>
               </button>
             )}
+            {this.showPasswordToggle && !readonly && !disabled && (
+              <button
+                aria-label="show password"
+                aria-checked={type === 'text' ? 'true' : 'false'}
+                aria-controls={this.inputId}
```

Comment:
```suggestion
                aria-controls={inputId}
```

We've already deconstructed this before, so we can access the local variable instead.

Reply:
Done!